java/connector-node/risingwave-sink-es-7/src/main/java/com/risingwave/connector/EsSink.java

```diff
@@ -154,11 +226,16 @@ private BulkProcessor.Builder applyBulkConfig(
 
     private BulkProcessor createBulkProcessor() {
```

Comment:
nits: it's better to pass the request tracker from the parameter list instead of using `this.requestTracker`, because this method is used in class initialization, and it will implicitly depend on that this method is called after `this.requestTracker` is initialized. Though current implementation is correct, but may easily cause error in the future.